proxy/example-local-fronend.js

```diff
@@ -0,0 +1,14 @@
+/*global module, process*/
+
+// Hack so that Mac OSX docker can sub in host.docker.internal instead of localhost
+// see https://docs.docker.com/docker-for-mac/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host
+const localhost = (process.env.PLATFORM === 'linux') ? 'localhost' : 'host.docker.internal';
+
+module.exports = {
+    routes: {
+        '/insights/remediations': { host: `https://${localhost}:8002` },
```

Comment:
`remediations` and `inventory` will be on different port because you'd run them from different webpack.